test/utils/status.ts

```diff
@@ -0,0 +1,15 @@
+import nock from 'nock'
+import { APP_CHECK_NAME } from '../../src/config'
+
+export function checkSuccessStatus(): void {
+  nock('https://api.github.com')
+    .post('/repos/tianhaoz95/approveman-test/check-runs', (body: any) => {
```

Comment:
Codacy found an issue: [Type declaration of 'any' loses type-safety. Consider replacing it with a more precise type.](https://app.codacy.com/manual/tianhaoz/approveman/pullRequest?prid=6083797)